tests/torch/test_graph_building.py

```diff
@@ -320,6 +372,39 @@ def test_split_attributes(input_shape):
                 assert chunk_nodes_with_attributes[node.node_name] is None
 
 
+class SplitByGetItemModel(ModelWithDummyParameter):
```

Comment:
TensorFlow has the same 'tf.__operators__.getitem' operation. More information you can find here https://www.tensorflow.org/api_docs/python/tf/Tensor#__getitem__. Are you going to collect  GetItemLayerAttributes for TensorFlow in this PR as well?

Reply:
will be covered separately in 106557